cloudify_cli/utils.py

```diff
@@ -228,6 +232,16 @@ def is_use_colors():
     return config.colors
 
 
+def get_import_resolver():
+    if not is_initialized():
+        return DefaultImportResolver()
```

Comment:
I don't think we want the default implemented at this level. I think it should be like "get_management_user"...